tools/Makefile

```diff
@@ -319,7 +319,7 @@ caml_tex := caml-tex$(EXE)
 
 $(caml_tex): INCLUDES += $(addprefix -I $(ROOTDIR)/otherlibs/,str $(UNIXLIB))
 $(caml_tex): $(caml_tex_files)
-	$(ROOTDIR)/runtime/ocamlrun$(EXE) $(ROOTDIR)/ocamlc$(EXE) -nostdlib \
+	$(OCAMLRUN) $(ROOTDIR)/ocamlc$(EXE) -nostdlib \
```

Comment:
Likewise, this is correct but possibly worth a comment clarifying (especially if what I write is wrong!). `$(ROOTDIR)/ocamlc$(EXE)` _runs_ on `$(OCAMLRUN)` but produces code for `$(ROOTDIR)/runtime/ocamlrun$(EXE)` and so although it's doing `-I $(ROOTDIR)/stdlib`, it still uses `$(OCAMLRUN)`.